packages/muelu/test/scaling/MatrixLoad.hpp

```diff
@@ -296,6 +296,7 @@ void readUserBlks(const std::string& userBlkFileName, const std::string& smoothe
           if (mueluList.sublist(smootherOrCoarse + ": params").sublist("subdomain solver parameters").get<std::string>("partitioner: type") == "user") {
 
             FILE   *fp;
+	    int    retVal;
```

Comment:
not sure if github is playing tricks on me here: indentation?

Reply:
my emacs config might be messed up on this machine. how do I keep the default emacs from inserting tabs?

Reply:
I have `(setq-default indent-tabs-mode nil)` in my config.

Reply:
okay it should be fixed now. thanks for that :)